wolfssl/wolfcrypt/wc_port.h

```diff
@@ -676,14 +676,26 @@ WOLFSSL_API int wolfCrypt_Cleanup(void);
     #define XFGETS     fgets
     #define XFPRINTF   fprintf
 
-    #if !defined(USE_WINDOWS_API) && !defined(NO_WOLFSSL_DIR)\
+    #if !defined(NO_WOLFSSL_DIR)\
         && !defined(WOLFSSL_NUCLEUS) && !defined(WOLFSSL_NUCLEUS_1_2)
+    #if defined(USE_WINDOWS_API)
+        #include <sys/stat.h>
+        #define XSTAT       _stat
+        #define SEPARATOR_CHAR ';'
```

Comment:
I don't see SEPARATOR_CHAR defined for WOLFSSL_ZEPHYR or WOLFSSL_TELIT_M2MB in the elif sections below, although it looks like it is used without protections for these platforms in ssl.c.

Reply:
Added the same SEPARATOR_CHAR as defined in else section.